arches/app/models/migrations/7787_relational_data_model.py

```diff
@@ -4,7 +4,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ("models", "7128_resource_instance_filter"),
+        ("models", "7784_add_serialized_graph_to_graph_publications"),
```

Comment:
I don't think that we want to change this migration, because it is going in 6.1, so this migration will already be executed well before `7784_add_serialized_graph_to_graph_publications` -- if you need this locally and already migrated, you'll need to reverse migrate and then re-run migrations

Reply:
ah -- fair and smart